doc/input-output-reference/src/overview/group-radiative-convective-units.tex

```diff
@@ -2063,3 +2069,5 @@ \subsubsection{Inputs}\label{inputs-11-013}
       Z5VentSlabIn,            !- Slab In Node Name for Surface 3
       Z5VentSlabOut;           !- Slab Outlet Node Name for Surface 3
 \end{lstlisting}
+\]
+\end{document}
```

Comment:
LaTexiT is probably adding it.  There may be an option to prevent it or you may need to diff the file after editing and remove any extra changes like this.